CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Change Log
 
+## [1.17.0](https://github.com/openeuropa/oe_bootstrap_theme/tree/1.17.0) (2024-11-06)
+[Full Changelog](https://github.com/openeuropa/oe_bootstrap_theme/compare/1.16.0...1.17.0)
+
+**Merged pull requests:**
+
+- OEL-3421: Added accessible toggle for modal and offcanvas. [\#444](https://github.com/openeuropa/oe_bootstrap_theme/pull/444) ([tibi2303](https://github.com/tibi2303))
+- OEL-3417: Added subtitle tag for featured media. [\#438](https://github.com/openeuropa/oe_bootstrap_theme/pull/438) ([tibi2303](https://github.com/tibi2303))
+
 ## [1.16.0](https://github.com/openeuropa/oe_bootstrap_theme/tree/1.16.0) (2024-10-08)
 [Full Changelog](https://github.com/openeuropa/oe_bootstrap_theme/compare/1.15.0...1.16.0)
```

Comment:
a PR is missing a bit more down with the PR for preparing of the previous release

Reply:
done.